Translation: Hindi (hi) Language

AkhileshThite opened the pull request:
### Description
I've added Hindi language translation to this website. Hindi is national language of India spoken by 341 million people around the world.

### Motivation and Context
I love Dogecoin! because of its beautiful community and simplicity. Currently Dogecoin is boom in the market and Indians would love to have our national language on the official website of one of the top cryptocurrencies in the world!

### How Has This Been Tested?
- I followed the steps in OSX Guide.
- Made hi.json
- Made all the changes accordingly.

Please check the screenshots attached below 👇
Thank you :)

### Screenshots (if appropriate):
![Screenshot 2021-05-10 at 7 58 41 PM](https://user-images.githubusercontent.com/68826419/117675929-ae41f380-b1ca-11eb-8fc6-3e1d89a7b81c.png)
![Screenshot 2021-05-10 at 7 58 56 PM](https://user-images.githubusercontent.com/68826419/117675937-b00bb700-b1ca-11eb-8ee2-2b5fa5228dd1.png)
![Screenshot 2021-05-10 at 7 59 05 PM](https://user-images.githubusercontent.com/68826419/117675944-b1d57a80-b1ca-11eb-96ff-e4893bbdc4fd.png)
![Screenshot 2021-05-10 at 7 59 58 PM](https://user-images.githubusercontent.com/68826419/117675963-b6019800-b1ca-11eb-9902-dbffacb49cf8.png)


### Checklist:
- [x] The necessary translations have been added for all languages

langerhans commented on .idea/vcs.xml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Please remove this whole directory.

akhileshthite replied:
Done.

langerhans commented on index.html:
```diff
@@ -268,7 +268,7 @@ <h2 data-i18n="dgc-getting-started" class="page-section-heading text-center text
                           <li><a class="dropdown-item select-lang" data-lang="de">Deutsch</a></li>
                           <li><a class="dropdown-item select-lang" data-lang="es">Español</a></li>
                           <li><a class="dropdown-item select-lang" data-lang="fr">Français</a></li>
-                          <li><a class="dropdown-item select-lang" data-lang="it">Italiano</a></li>
```
Please undo this deletion :)

akhileshthite replied:
Okay, Sorry for that I forgot.

akhileshthite replied:
@langerhans Done.

langerhans replied:
Would appreciate if you fixed the indentation on the line again so it looks clean. Sorry for nitpicking.

akhileshthite replied:
Done.
Sir It's okay! you can ask for any changes :)

langerhans reviewed (dismissed):
I'll leave this open for a bit longer. Maybe someone else who speaks the language can review the changes as well :)

akhileshthite commented:
> I'll leave this open for a bit longer. Maybe someone else who speaks the language can review the changes as well :)

Okay sir :)

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-android-2-0": "2. वॉलेट लॉन्च करें।",
+    "dgc-getting-started-android-2-1": "उसी पृष्ठ पर, \"Open\" बटन दबाएं। आपको डॉगकोइन वॉलेट में निर्देशित किया जाएगा।",
+    "dgc-getting-started-android-2-2": "तुम भी अपने Android ™ स्मार्टफोन पर एप्लिकेशन दराज के भीतर Dogecoin बटुआ तक पहुंच सकते हैं।",
+    "dgc-getting-started-android-3-1": "अब आप अपने हाथ में एक पूर्ण Dogecoin wallet है!",
```
```suggestion
    "dgc-getting-started-android-3-1": "अब aapke(hindi) हाथ में एक पूर्ण Dogecoin wallet है!",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-web-blockio-3-1": "एक सुरक्षित स्थान पर उत्पन्न गुप्त Mnemonic रखें।",
+    "dgc-getting-started-web-blockio-3-2": "यह पुष्टि करने के लिए बॉक्स की जांच करें कि क्या आप जानते हैं कि यदि आप अपना पिन और मेनोनिक खो देते हैं, तो आप अपने सिक्कों तक पहुंच खो देंगे। <br/> किए गए बटन पर क्लिक करें।",
+    "dgc-getting-started-web-blockio-4-0": "4. अपने वॉलेट का उपयोग करें!",
+    "dgc-getting-started-web-blockio-4-1": "अब आपके पास Block.io पर एक ऑनलाइन डॉगकोइन वॉलेट है! <br/> Dogecoins भेजने और प्राप्त करने के लिए आप ऑनलाइन वॉलेट का उपयोग कर सकते हैं।",
```
```suggestion
    "dgc-getting-started-web-blockio-4-1": "अब आपके पास Block.io पर एक ऑनलाइन Dogecoin वॉलेट है! <br/> Dogecoins भेजने और प्राप्त करने के लिए आप ऑनलाइन वॉलेट का उपयोग कर सकते हैं।",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-web-blockio-4-1": "अब आपके पास Block.io पर एक ऑनलाइन डॉगकोइन वॉलेट है! <br/> Dogecoins भेजने और प्राप्त करने के लिए आप ऑनलाइन वॉलेट का उपयोग कर सकते हैं।",
+    "dgc-getting-started-web-blockio-4-2": "Dogecoins जमा करने के लिए डिपॉजिट बटन पर क्लिक करें, Dogecoins वापस लेने के लिए Withdraw बटन पर क्लिक करें।",
+    "dgc-getting-started-web-blockio-acknowledge": "तस्वीरें, Block.io गाइड, और वेबसाइट made by <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-paper-note": "<strong> ध्यान दें: </ strong> डॉगकोइन के दीर्घकालिक भंडारण के लिए, एक अच्छे सुरक्षा अभ्यास के रूप में पेपर पर्स की अत्यधिक अनुशंसा की जाती है।",
```
```suggestion
    "dgc-getting-started-paper-note": "<strong> ध्यान दें: </ strong> Dogecoin के दीर्घकालिक भंडारण के लिए, एक अच्छे सुरक्षा अभ्यास के रूप में पेपर wallet(in hindi) की अत्यधिक अनुशंसा की जाती है।",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-web-warning": "<Strong> नोट: </ strong> Dogecoin की बड़ी मात्रा में भंडारण के लिए, डेस्कटॉप पर्स अत्यधिक एक अच्छा सुरक्षा अभ्यास के रूप में सिफारिश की है।",
+    "dgc-getting-started-web-blockio-1-0": "1. Block.io पर साइन अप करें",
+    "dgc-getting-started-web-blockio-1-1": "पंजीकरण पृष्ठ पर जाने के लिए नीचे दिए गए लिंक पर क्लिक करें। <br/> ईमेल और पासवर्ड भरें, फिर साइन अप बटन पर क्लिक करें।",
+    "dgc-getting-started-web-blockio-1-2": "एक Block.io खाता यहां साइन इन करें।",
```
English file uses **signup** not sign in  L221

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-android-1-3": "एप्लिकेशन डाउनलोड और इंस्टॉल करने के लिए प्रतीक्षा करें।",
+    "dgc-getting-started-android-2-0": "2. वॉलेट लॉन्च करें।",
+    "dgc-getting-started-android-2-1": "उसी पृष्ठ पर, \"Open\" बटन दबाएं। आपको डॉगकोइन वॉलेट में निर्देशित किया जाएगा।",
+    "dgc-getting-started-android-2-2": "तुम भी अपने Android ™ स्मार्टफोन पर एप्लिकेशन दराज के भीतर Dogecoin बटुआ तक पहुंच सकते हैं।",
```
```suggestion
    "dgc-getting-started-android-2-2": "तुम भी अपने Android ™ स्मार्टफोन पर एप्लिकेशन दराज के भीतर Dogecoin wallet(in hindi) तक पहुंच सकते हैं।",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-linux-3-2": "ठीक है, हमने लिनक्स पर 1, 2, 3 के बारे में झूठ बोला होगा। लेकिन, हमें अन्य ऑपरेटिंग सिस्टम की तुलना में अधिक समझाने की आवश्यकता है। माफ़ करना!",
+    "dgc-getting-started-linux-4-0": "4. वॉलेट इंस्टॉल करें।",
+    "dgc-getting-started-linux-4-1": "स्थापना प्रक्रिया से गुजरें। एक बार जब आप लाइसेंस समझौते में पहुंच जाते हैं, तो इसे स्थापना के साथ जारी रखना स्वीकार करें।",
+    "dgc-getting-started-linux-4-2": "एक बार जब आप स्थापना के साथ समाप्त कर लें, MultiDoge शॉर्टकट खोजने के लिए और इसे खोलने।",
```
```suggestion
    "dgc-getting-started-linux-4-2": "एक बार जब आप स्थापना के साथ समाप्त कर लें, MultiDoge शॉर्टकट khoje aur usse khole(in hindi) ।",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-osx-2-1": "फ़ाइल पर क्लिक करने पर एक नई विंडो खुलनी चाहिए। क्लिक करें और \"Applications\" फ़ोल्डर में \"MultiDoge\" आइकन खींचें।",
+    "dgc-getting-started-osx-2-2": "यह wallet को आपके एप्लिकेशन मेनू में ले जाएगा।",
+    "dgc-getting-started-osx-acknowledge": "फ़ोटो और macOS गाइड by <a href=\"https://www.reddit.com/user/FlippyDog101\" target=\"_blank\">/u/FlippyDog101</a>. वेबसाइट बनाई by <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-linux-1-1": "स्थानीय डॉगकोइन वॉलेट MultiDoge डाउनलोड करने के लिए नीचे दिए गए लिंक पर क्लिक करें। <br/> फ़ाइल डाउनलोड करने के लिए फ़ाइल सहेजें।",
```
```suggestion
    "dgc-getting-started-linux-1-1": "स्थानीय Dogecoin वॉलेट MultiDoge डाउनलोड करने के लिए नीचे दिए गए लिंक पर क्लिक करें। <br/> फ़ाइल डाउनलोड करने के लिए फ़ाइल save kare(in hindi)।",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-getting-started-walletgen": "WalletGenerator",
+    "dgc-getting-started-paper-choose": "कृपया अपना पेपर वॉलेट चुनें।",
+    "dgc-getting-started-1-0": "1. वॉलेट डाउनलोड करें।",
+    "dgc-getting-started-windows-1-1": "स्थानीय डॉगकोइन वॉलेट MultiDoge डाउनलोड करने के लिए नीचे दिए गए लिंक पर क्लिक करें। <br/> यदि आपको डाउनलोड स्वीकार करने के लिए कहा जाता है, तो डाउनलोड स्वीकार करें।",
```
```suggestion
    "dgc-getting-started-windows-1-1": "स्थानीय Dogecoin वॉलेट MultiDoge डाउनलोड करने के लिए नीचे दिए गए लिंक पर क्लिक करें। <br/> यदि आपको डाउनलोड स्वीकार करने के लिए कहा जाता है, तो डाउनलोड स्वीकार करें।",
```

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-osx-wallet-modal-dogecoin-core": "Dogecoin कोर।",
+	"dgc-android-wallet-modal-title": "Google Play से डाउनलोड करें।",
+    "dgc-android-wallet-modal-apk": "<a href=\"https://github.com/langerhans/dogecoin-wallet-new/releases/download/1.08/wallet-1.08-aligned.apk\">प्रत्यक्ष APK डाउनलोड।</a>",
+	"dgc-ios-wallet-modal-title": "App स्टोर से डाउनलोड करें।",
```
Since you left google play in english completely it's advisable to keep app store in english completely too

akhileshthite replied:
Done

vj-codes commented on assets/i18n/hi.json:
```diff
@@ -0,0 +1,267 @@
+    "dgc-windows-wallet-modal-sync": "Dogecoin का उपयोग करने के लिए आपको ब्लॉकचेन के साथ सिंक करना होगा।",
+    "dgc-windows-wallet-modal-ledger": "इस का मतलब है, आप Dogecoin की जरूरत \"ledger\" to use Dogecoin.",
+    "dgc-windows-wallet-modal-light-wallet": "Multidoge एक \"Light\" wallet है। यह \"Skimming\" द्वारा blockchain साथ समन्वयित हो जाता blockchain के माध्यम से, तेजी से सिंक बार प्रदान करते हैं।",
+    "dgc-windows-wallet-modal-full-wallet": "दूसरी ओर, डॉगकोइन कोर एक \"Full\" wallet है। यह इसे डाउनलोड करने, एक ठोस काम कर Dogecoin बटुआ प्रदान करके समन्वयन करती है।",
```
```suggestion
    "dgc-windows-wallet-modal-full-wallet": "दूसरी ओर, Dogecoin core एक \"Full\" wallet(write wallet in hindi here) है। यह इसे डाउनलोड करने, एक ठोस काम कर Dogecoin बटुआ(wallet in hindi) प्रदान करके समन्वयन करती है।",
```

akhileshthite replied:
Done

vj-codes reviewed (changes requested):
@AkhileshThite hey I suggested some changes just minor typos let me know what you think about them

akhileshthite commented:
> @AkhileshThite hey I suggested some changes just minor typos let me know what you think about them

Thanks! I'll fix them up quickly :)

akhileshthite commented:
@langerhans Sir, I fixed all the minor typos reviewed by @vj-codes.

langerhans commented:
Thank you!

akhileshthite commented:
> Thank you!

Thank you so much!
I would love to contribute more.

ajaygalagali commented:
> I've added Hindi language translation to this website. Hindi is national language of India spoken by 341 million people around the world.


For your information - Hindi isn't the national language of India. India has no national language. Check [here][wiki].  Please don't spread incorrect information.

[wiki]:https://en.wikipedia.org/wiki/National_language#India

akhileshthite commented:
> > I've added Hindi language translation to this website. Hindi is national language of India spoken by 341 million people around the world.
> 
> For your information - Hindi isn't the national language of India. India has no national language. Check [here](https://en.wikipedia.org/wiki/National_language#India). Please don't spread incorrect information.

@ajaygalagali, well that's what "I was taught by my school" (was in the syllabus) probably 8-10 years ago, now "Wikipedia" and "google" are saying no national language. 
If a person in India doesn't understand any other language, then most of the time he or she will understand "Hindi" and yes it is spoken by 341 million people around the world.
Anyways I've corrected my PR comment, because google says India does not have any national language.
I don't want to argue here on the professional platform, and honestly don't want to waste my time over this.
"I've corrected my PR comment anyways".
Thank you!

vj-codes commented:
@AkhileshThite it's not only you who's been taught like that , I'm still in high school (from Maharastra) and we are still taught the same , it's the system that does interpret the difference between official and national 
@adithyapaib thank you for sharing!
